elegant/clean_timepoint_data.py

```diff
@@ -113,11 +122,32 @@ def remove_excluded_positions(experiment_root,dry_run=False):
     annotations = load_data.read_annotations(experiment_root)
     good_annotations = load_data.filter_annotations(annotations, load_data.filter_excluded)
     excluded_positions = sorted(set(annotations.keys()).difference(set(good_annotations.keys())))
+
+    excluded_root = experiment_root / 'excluded_positions'
+
     for position in excluded_positions:
         if (experiment_root / position).exists():
             print(f'Found an excluded position to delete {position}')
             if not dry_run:
-                shutil.rmtree(str(experiment_root / position))
+                (excluded_root / position).mkdir(parents=True,exist_ok=True)
+                (excluded_root / 'annotations').mkdir(parents=True,exist_ok=True)
+
+                shutil.copy(experiment_root / position / 'position_metadata.json',
+                    excluded_root / position / 'position_metadata.json')
+                shutil.copy(experiment_root / 'annotations' / f'{position}.pickle',
+                    excluded_root / 'annotations' / f'{position}.pickle')
+                shutil.copy(experiment_root /  'experiment_metadata.json',
```

Comment:
What about just not copying it if it's already there? That way the original version is preserved, even if the function gets run several times...

Reply:
Good call. Implemented that strategy.